docs/topics/simulation_setup.rst

```diff
@@ -0,0 +1,182 @@
+simulation), path sampling requires an initial trajectory and a description
+of the path ensemble. However, both describing the ensemble and creating an
+initial trajectory are more difficult in path sampling than the equivalent
+problems in configurational sampling.
+
+This document discusses the general idea of setting up path sampling
+simulations. In includes a few references to specifics of OPS, but is mainly
+focused on the concepts. The examples section of the website includes many
+practical uses of these ideas.
+
+-----
+
+Defining the ensemble
+=====================
+
+The nature of the ensemble depends on the specific algorithm being used.
+Here, we'll discuss ensembles for TPS and TIS. In OPS, the :ref:`transition
+network <transitions-and-networks>` is used to efficiently create *all* the
+ensembles to be used in a simulation. For TPS, there is only one ensemble to
+be sampled. For TIS, there is one ensemble per interface (plus additional
+helper ensembles, such as the minus ensemble). In multiple state systems,
+TIS can involve hundreds of ensembles.
+
+Monte Carlo methods (configurational or path sampling) draw samples from a
+statistical ensemble. For configurational sampling, the thermodynamic
+ensemble to be sampled is defined by the underlying Hamiltonian of the
+system (which includes the masses of the particles and the interaction
+energies) and the natural variables of the ensemble (e.g., for the
+canonical, or :math:`NVT`, ensemble, these are the number of particles, the
+volume, and the temperature).  Similarly, a path ensemble will be defined by
+the underlying dynamics (including the underlying thermodynamic ensemble)
+and additional restrictions (such as state definitions).
+
+Selecting the extra restrictions for path sampling is much more difficult
+that selecting the thermodynamic state to sample: whereas the temperature
+and density usually come from experimental considerations, the volume of
+phase space that defines a stable state is not an experimentally observable
+quantity. Typically, defining the state requires performing MD simulations
+in each stable state to characterize its bounds, and there is not a unique
+appropriate definition when studying a particular problem of interest.
+
+In TPS, only the states need to be defined. In TIS, one must also define the
+interfaces.
+
+Defining good states
+--------------------
+
+In TPS, the main traits defining the path ensemble are the state
+definitions. The essential point is that states should be stable: a
+trajectory started from any point in the state should be expected to take a
+long time before visiting any other state.
+
+The topic of state definitions has been included in several review articles.
+For more, see:
+
+* `Bolhuis and Dellago. "Trajectory-based rare event simulations." in
+  Reviews Computation Chemistry vol. 27 (2010)
+  <http://onlinelibrary.wiley.com/doi/10.1002/9780470890905.ch3/summary>`_
+
```

Comment:
Practical and conceptual path sampling issues
PG Bolhuis, C Dellago
The European Physical Journal Special Topics 224 (12), 2409-2427